.github/workflows/go.yml

```diff
@@ -203,6 +203,11 @@ jobs:
     if: needs.detect-noop.outputs.noop != 'true'
 
     steps:
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
```

Comment:
why do we need to setup go here?

Reply:
Ubuntu 20.04 pre-installed go v1.15, that's why in previous CI, task using Makefile/staticcheck failed but CI/staticcheck succeed.

Reply:
`go get` in v1.15 will get the latest released tag but `go get` in v1.16 will get the latest master. I just make CI work consistently with Makefile.

Reply:
cool, it's better